workspaces/scanner/src/types.ts

```diff
@@ -218,6 +219,11 @@ export interface Payload {
   };
 }
 
+export type SemverRange = string;
+export type Spec = string;
+
+export type HighlightPackages = Record<Spec, SemverRange | "*">;
```

Comment:
Was more for the example, also Spec was not correct since it's only the name
```suggestion
export type SemverRange = string | "*";

export type HighlightPackages = Record<string, SemverRange>;
```

WDYT?